jekyll/_cci2/server/installation/phase-1-prerequisites.adoc

```diff
@@ -435,7 +432,7 @@ CircleCI server listens on a number of ports which need to be configured dependi
 * Nomad server [TCP 4647]
 * Output processor [gRPC 8585]
 
-Depending on your requirements you may choose to terminate TLS for only the frontend/api-gateway or provide TLS for services listening on all the ports. 
+Depending on your requirements you may choose to terminate TLS for only the frontend/api-gateway or provide TLS for services listening on all the ports.
```

Comment:
Don't know if it's worth mentioning here, but TLS can't be terminated on this port: 4647 when using ACM, since it's already encrypted by nomad's mTLS.

Reply:
this whitespace correction happened automatically on save, my ticket does not address this particular part of the docs.  Does it make sense to call it out of scope and ticket it out separately if there is something to fix?